conf/messages

```diff
@@ -141,3 +143,27 @@ pensions.reliefAtSourceTotalPaymentsAndTaxReliefAmount.whereToFind.agent = You c
 pensions.reliefAtSourceTotalPaymentsAndTaxReliefAmount.howToWorkOut = To work it out yourself, divide the amount you actually paid by 80 and multiply the result by 100.
 pensions.reliefAtSourceTotalPaymentsAndTaxReliefAmount.calculationHeading = Example calculation
 pensions.reliefAtSourceTotalPaymentsAndTaxReliefAmount.exampleCalculation = Emma paid £500 into her pension scheme. £500 divided by 80 and multiplied by 100 is £625. Her answer is £625.
+
+#payments into pensions
+
+paymentsIntoPensions.cya.heading.individual = Check your payments into pensions
+paymentsIntoPensions.cya.heading.agent = Check your client’s payments into pensions
+paymentsIntoPensions.cya.reliefAtSource = Relief at source (RAS) pension payments
+paymentsIntoPensions.cya.reliefAtSourceTotal = Total RAS payments plus tax relief
+paymentsIntoPensions.cya.oneOff = One-off RAS payments
+paymentsIntoPensions.cya.oneOffTotal = Total one-off RAS payments plus tax relief
+paymentsIntoPensions.cya.pensionsTaxReliefNotClaimed = Pensions where tax relief is not claimed
+paymentsIntoPensions.cya.retirementAnnuity = Retirement annuity contract payments
+paymentsIntoPensions.cya.retirementAnnuityTotal = Total retirement annuity contract payments
+paymentsIntoPensions.cya.workplacePayments = Workplace pension payments
+paymentsIntoPensions.cya.workplacePaymentsTotal = Total workplace pension payments
+
+paymentsIntoPensions.cya.reliefAtSource.hidden = Change whether relief at source pensions payments were made
+paymentsIntoPensions.cya.reliefAtSourceTotal.hidden = Change total relief at source pensions payments, plus tax relief
+paymentsIntoPensions.cya.oneOff.hidden = Change whether one-off relief at source pensions payments were made
+paymentsIntoPensions.cya.oneOffTotal.hidden = Change total one-off relief at source pensions payments, plus tax relief
+paymentsIntoPensions.cya.pensionsTaxReliefNotClaimed.hidden = Change whether payments were made into a pension where tax relief was not claimed
```

Comment:
_pensionsTaxReliefNotClaimed.hidden_
This hidden text looks  ok to me but where did it come from as it's omitted from the hidden text document in the Jira Ticket. Can we confirm this is correct (if we already have could we update the document)?
It's different to the prototype hidden text otherwise.